<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreferencesDialog</class>
 <widget class="QDialog" name="PreferencesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>137</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/icons/icons/application-default-icon.png</normaloff>:/icons/icons/application-default-icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupADB">
     <property name="title">
      <string>ADB location</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit_path">
          <property name="toolTip">
           <string>Directory where the adb command is found (probably the SDK's platform-tools directory)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_browse">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="cbo_adbOnPath">
        <property name="toolTip">
         <string>Check if your PATH is configured to include the SDK platform tools</string>
        </property>
        <property name="text">
         <string>ADB is on PATH</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_path</tabstop>
  <tabstop>button_browse</tabstop>
  <tabstop>cbo_adbOnPath</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbo_adbOnPath</sender>
   <signal>clicked()</signal>
   <receiver>PreferencesDialog</receiver>
   <slot>setDirConfigState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>62</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>-5</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setDirConfigState()</slot>
 </slots>
</ui>
